src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -746,6 +751,10 @@ internal override void DecodeWellKnownAttribute(ref DecodeWellKnownAttributeArgu
                 arguments.GetOrCreateData<ParameterWellKnownAttributeData>().HasEnumeratorCancellationAttribute = true;
                 ValidateCancellationTokenAttribute(arguments.AttributeSyntaxOpt, (BindingDiagnosticBag)arguments.Diagnostics);
             }
+            else if (attribute.IsTargetAttribute(this, AttributeDescription.InterpolatedStringHandlerArgumentAttribute))
```

Comment:
>IsTargetAttribute [](http://example.com/codeflow?start=31&length=17)

Since we need to deal with two different signatures here, I think we should use GetTargetAttributeSignatureIndex directly rather than trying to infer what signature we are dealing with based on values.